### ISSUE TITLE
Don't work anymore on the last Insider Build

### ISSUE THREAD

**Issue by Brouilles** [Simnico99/MicaWPF#1]
Microsoft  has disabled the trick to apply Mica with dwmapi. In the last Insider Build.
Only solution now seems to recreate it like https://github.com/ad2017gd/Mica32.

**Comment by Simnico99** [comment]
Yep I am looking into doing something like that.

**Comment by Brouilles** [comment]
I have done an implementation. But [Airspace Issue](https://github.com/dotnet/wpf/issues/152), a crazy thing. For the moment I have abandoned.
If you want here the code. WPF is render behind the compositor. I don't find a solution. 

Two exist but not very good:

1. Overlapping Windows:
You can create multiple WPF windows one for rendereing the WPF Content and one for Acrylic Effect, These windows should be transparent and must communicate with each other to sync the window resize, window drag and other events, this is possible by overriding WndProc. But this method will cause Flickerng on resize and also affects the Acrylic Effect during maximize animation. (This has been tested before)

2. WPF Swapchain Hack:
This is another possible way that i haven't tested yet, but theoretically it might be working. This can be done by hacking into WPF Swapchain to obtain its back buffer and copy it into an ID2D1Bitmap or similar and then passing it to the Windows.UI.Composition.Visual by using somethng like Composition Native Interoperation. By this way we may be able to overcome the Airspace Issue with WPF Acrylic Effect. (😅 The truth is Currently i was only able to get the swapchain backbuffer from WPF)

3. Convert the composer Sprite to a Win2D buffer, to convert it to a WPF bitmapImage or similar. But I am stuck at the first step.

Here the code if you want
[InteropCompositor-master.zip](https://github.com/Simnico99/MicaWPF/files/7566346/InteropCompositor-master.zip)

**Comment by Brouilles** [comment]
GDI+ solution can also work. But some problem with the current implementation for the moment (with multiple monitor) https://github.com/ad2017gd/Mica32/issues/3

But don't know if it is possible to pass the hwnd of WPF to a C++ dll. I think yes in theory.

**Comment by Simnico99** [comment]
Well,
I've worked on a wrapper for Mica32 it actually works I've been able to pass the HWND of the WPF window.
It does work but my only issues right now is that it makes a rectangle that delete the controls in the Window.
I'd be trying to modify that to return a brush or something, We will see.


If you wanna take a look I've pushed a branch with the Mica32 and Mica32Wrapper:
https://github.com/Simnico99/MicaWPF/tree/Mica32-dev-test
